server/routes/newspaperArchive.ts

```diff
@@ -69,3 +80,28 @@ router.get('/auth', async (req: Request, res: Response) => {
 });
 
 export { router };
+
+async function checkSupporterEntitlement(req: Request): Promise<boolean> {
+	const supporterAttributesResponse = await getSupporterStatus(req);
+	const supporterAttributes = await supporterAttributesResponse.json();
+
+	// ToDo: this should return a flag that represents either Tier 3 or a newspaperArchive specific entitlement
+	return (
+		supporterAttributes.contentAccess['guardianWeeklySubscriber'] &&
+		supporterAttributes.contentAccess['supporterPlus']
+	);
+}
+
+async function getSupporterStatus(req: Request) {
+	const host = 'members-data-api.' + conf.DOMAIN;
+
+	return fetch(`https://${host}/user-attributes/me`, {
+		method: 'GET',
+		headers: {
+			...(await authorizationOrCookieHeader({ req, host })),
+			'Content-Type': 'application/json',
```

Comment:
Super minor point - but do we need to send a `Content-Type` header here? My understanding is that it's needed as a request header when sending a request with a particular content type in the body (which isn't the case here as it's a GET).

Reply:
I think you're right! Was copied from the other code that calls mdapi but doesn't seem necessary here. I've also copied the `X_GU_ID_FORWARDED_SCOPE` which I'm assuming is important. I'll try test without it...